package.json

```diff
@@ -65,4 +65,4 @@
     "webpack": "^1.14.0",
     "yargs": "^6.6.0"
   }
-}
+}
```

Comment:
Nope, not at all. I had some problems using git and had to copy paste the package.json. Besides, I don't understand why there are 74 commits mostly from others associated with this PR.